src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarBidAction.tsx

```diff
@@ -107,18 +107,16 @@ export class ArtworkSidebarBidAction extends React.Component<
               Register to bid
             </Button>
           )}
-          {registrationAttempted &&
-            !registeredToBid && (
-              <Button width="100%" size="large" mt={1} disabled>
-                Registration pending
-              </Button>
-            )}
-          {registrationAttempted &&
-            registeredToBid && (
-              <Button width="100%" size="large" mt={1} disabled>
-                Registration complete
-              </Button>
-            )}
+          {registrationAttempted && !registeredToBid && (
```

Comment:
Interesting that this is a change -- it looks like prettier updated it's formatter style.